src/pages/elements/app-icons.mdx

```diff
@@ -9,3 +9,1172 @@ They allow users to identify apps at a glance while serving as a unique
 expression of our brand.
 
 </PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Overview</AnchorLink>
+  <AnchorLink>Stroke icons</AnchorLink>
+  <AnchorLink>Fill icons</AnchorLink>
+  <AnchorLink>IBM Plex icons</AnchorLink>
+  <AnchorLink>Hero brands and 3rd parties</AnchorLink>
+  <AnchorLink>Usage</AnchorLink>
+</AnchorLinks>
+
+## Resources
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="App icon builder (.ai)"
+      aspectRatio="2:1"
+      href="/files/App_Icon_Builder.ai"
+      actionIcon="download"
+      >
+
+![](/images/ai.svg)
+
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="App icon template (.ai)"
+      aspectRatio="2:1"
+      href="https://github.ibm.com/carbon/app-icons/tree/master/app-icon-template"
```

Comment:
Hey! I noticed this was pointing to a repo instead of a download but it looks like a download, should that be changed? It's also like that on the original site so it might be all good!

Reply:
I updated these, I was just copying what the live site had, but you're right, it isn't a download link

Reply:
Did we decide to make it a download as well?

Reply:
@vpicone the first card is actually a download to an ai file we are hosting, the second card is a link to the github repo.